### PR TITLE
Robert Gruener_HW1 Submission

### PR DESCRIPTION
**Pull request by RFGruener**
HW1 submission for Robert Gruener... in reality, HW1 submission attempt. 

Although no coding was required for this submission, this is one of the most difficult and frustrating things I've done in a while. I did follow the steps in the General Homework Guidelines, but still had some difficulty in knowing the order of operations and what exactly procedure. Indeed at first I felt lost in the sea of information on the General Homework Guidelines, links links everywhere but not a drop to drink.

 An initial issue I had was having git installed on my computer, that was east to take care of. But I still don't know how or if Git is associated with Github and the relationship between the two. Once I found the correct buttons to press, forking the assignment and getting it into RStudio wasn't too difficult and I enjoyed actually writing the md file. I was able to commit, but when I tried to push the repository back I got a long error with this "Updates were rejected because the remote contains work that you do
hint: not have locally." it also suggested that I do a pull request first, which gave me this error "fatal: refusing to merge unrelated histories." Which seems bad. 

I did some googling and couldn't figure out a solution. So I went back and started a new R project with the Github link, called it hw01_take2, copied and pasted my changes to the Readme.md files from the original one, committed, pulled, pushed and it worked. Shrug emoji, but I'm just happy to get it submitted.

### CODE REVIEW

**Comment by abbygirlrose** [comment]
Evaluation HW-01
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |    X       |              |            |
| **Ease of access**          |           |     X         |            |

Remarks:
- Nice use of formatting and detail!
- Consider using subject headers for easier reading
- Perhaps include a photo next time (although I do appreciate that you directed us where to find one on the lab site)
- on the line the starts with "####Err", you need a space after the hashtags to make the formatting occur

**Comment by marydeniserobertson** [comment]
Evaluation
----------------------------------------------------
| Topic                                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**                    |     x      |              |            |
| **Coding strategy**              |     x      |              |            |
| **Presentation**                    |             |    x        |            |
| **Achievement, creativity**  |     x      |              |            |
| **Ease of access**                |      x     |              |            |

Remarks:
Nice to meet you Robert! Thanks for the really detailed introduction. I found the process frustrating too, it was awesome when I finally managed to submit the homework! Just one piece of constructive criticism - I think you need to put a space between # and the text for it to appear as a heading, and a space after the * to get it to do a list. I had the exact same issue, and it displayed differently in R preview vs on Github, which I didn't realize.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
-# hw01
-Repository for homework 01
+## Hello World, it's me Robert!
+####Er, rather, hello CFSS TA and/or instructor and/or student, it's me Robert!
```
Comment (mayank-kgp) [inline README.md:2]: Space should have been given before Er

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+
+Currently, I am co-mentored in two labs. One is a "wet" lab where I work with cells and mice, and one is computational. I don't have a computational background, so I spend most of my time in the wet lab, but my advisors and I would like me to become more computational for various reasons. Hence this class. If you want to read more about my work, you can check out the [computational lab's website](http://huang-lab.umn.edu/meet-lab-members-0) and look for my bio (you can also see an *okay* pic of me). 
+
+I think that about wraps things up, but I'll go ahead and make a list of things I like doing in my free time just to make sure I get full credit for using different Markdown syntax:
+*Going to see musicals
+*Reading the occasional books (currently reading [severence](https://www.npr.org/2018/08/19/639251266/in-severance-the-world-ends-not-with-a-bang-but-a-memo))
+*Listening to podcasts (Freakonimcs, Dear Hank and John, and Reply All are some of my favs)
+*Going for runs (I can run half-marathons, but don't like signing up for races)
```
Comment (mayank-kgp) [inline README.md:16]: Give space after each * for converting it to a list item.

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |          |              |     x       |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |           |        x      |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* You have included all the required markdown elements. Overall, the page is good. Just do a self evaluation before submitting the work. You could have seen that list is not working.
* Git and GitHub difference - https://stackoverflow.com/a/13321586/4606981
* Git workflow might be confusing in the start. Just create a new issue [here](https://github.com/uc-cfss/hw01/issues) whenever your are stuck and not able to resolve anything.

**Pull request closed without merging**